attendance.php

```diff
@@ -55,6 +55,30 @@
 		PRIMARY KEY(id)
 	) DEFAULT CHARACTER SET=utf8mb4;
 
+	USE braven_attendance;
+	ALTER TABLE attendance_people
+	      DROP PRIMARY KEY,
+	      ADD dummy_id INT PRIMARY KEY AUTO_INCREMENT,
+	      ADD UNIQUE INDEX unique_index (`event_id`, `person_id`);
+
+	USE braven_attendance;
+	ALTER TABLE attendance_lc_absences
+	      DROP PRIMARY KEY,
+	      ADD dummy_id INT PRIMARY KEY AUTO_INCREMENT,
+	      ADD UNIQUE INDEX unique_index (`event_id`, `lc_email`);
+
+	USE wordpress;
```

Comment:
the wordpress stuff wouldn't be in attendance.php, this file is just for the attendance specific bit. so can remove this